lib/pages/host_scan_page.dart

```diff
@@ -18,16 +19,20 @@ class HostScanPage extends StatefulWidget {
 
 class _HostScanPageState extends State<HostScanPage>
     with TickerProviderStateMixin {
-  final Set<ActiveHost> _devices = {};
+  final Set<ActiveHost> _hosts = {};
   double _progress = 0;
   bool _isScanning = false;
   StreamSubscription<ActiveHost>? _streamSubscription;
+  late String? _ip;
```

Comment:
Change `ip` and `gatewayIp` to be of type `ActiveHost`

Reply:
I'm saving only the result of `NetworkInfo` methods, how can that be of type `ActiveHost` and not `String`

Reply:
I guess that we are running on our IP in another place, so it's ok.